pkg/cfn/builder/api_test.go

```diff
@@ -1551,7 +1551,7 @@ var _ = Describe("CloudFormation template builder API", func() {
 		})
 	})
 
-	Context("NodeGroup{EbsOptimized=true}", func() {
+	Context("NodeGroup{EbsOptimized=nil}", func() {
```

Comment:
We should add test for true and false cases, but I'm happy if you add those later on.

Reply:
It makes sense to add tests for this as it was a bug, but overall we can't keep adding tests for all possible values.